src/alpine/3.17/helix/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM amd64/alpine:3.17 AS msquic
+FROM alpine:3.17 AS msquic
```

All of these Dockerfiles will need to provide the explicit repo name. We have an [infrastructure issue](https://github.com/dotnet/docker-tools/issues/1341) that requires this.
```suggestion
FROM library/alpine:3.17 AS msquic
```